06-building-decays.md

```diff
@@ -149,8 +186,26 @@ from Configurables import DaVinci
 DaVinci().UserAlgorithms += [dstar_seq.sequence()]
 ```
 
+
+> ## Debugging your selection chain {.callout}
+> The `PhysSelPython.Wrappers` offers a very useful utility for debugging your selection chains, called `PrintSelection`.
+> It gets a `Selection` as input and it can be used the same way, except it will print the decay tree everytime making use of the `PrintDecayTree` algorithm which was discussed in the [Exploring a DST](05-interactive-dst.html) lesson.
+>
+> For more complex debugging, one can setup `DaVinci` with `graphviz
+> ```shell
+> SetupDaVinci --use "graphviz v* LCG_Interfaces"
```

Comment:
`lb-run/-dev` please unless it is not possible, in which case we need to add a note saying that it isn't possible yet but will be soon.